### PR TITLE
Add tests for DAQmxConfigureLogging and DAQmxStartNewFile

### PR DESCRIPTION
**Pull request by texasaggie97**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nidaqmx-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Add tests for `DAQmxConfigureLogging` and `DAQmxStartNewFile`.

### Why should this Pull Request be merged?

Increase test coverage

### What testing has been done?

New tests pass

### CODE REVIEW

**Review by zhindes — APPROVED** [review]

File: tests/acceptance/test_internationalization.py [hunk 1]
```diff
@@ -69,3 +70,63 @@ def test___supported_encoding___logging_file_path___returns_assigned_value(
     ai_task.in_stream.logging_file_path = file_path  # type: ignore[assignment] # https://github.com/ni/nidaqmx-python/issues/613
 
     assert ai_task.in_stream.logging_file_path == pathlib.Path(file_path)
+
+
+@pytest.mark.grpc_xfail(
+    reason="AB#2393811: DAQmxGetLoggingFilePath returns kErrorNULLPtr (-200604) when called from grpc-device.",
```
Comment (bkeryan) [inline tests/acceptance/test_internationalization.py:76]: FYI: This was fixed in March. Separate from this PR, we should update the version of DAQmx installed on nidaqmxbot and remove xfails referencing this bug.

File: tests/acceptance/test_internationalization.py [hunk 2]
```diff
@@ -69,3 +70,63 @@ def test___supported_encoding___logging_file_path___returns_assigned_value(
...
+        file_path,
+        logging_mode=expected_logging_mode,
+        group_name=expected_group_name,
+        operation=expected_logging_operation,
+    )
+
+    assert ai_task.in_stream.logging_file_path == pathlib.Path(file_path)
+    assert ai_task.in_stream.logging_mode == expected_logging_mode
+    assert ai_task.in_stream.logging_tdms_group_name == expected_group_name
+    assert ai_task.in_stream.logging_tdms_operation == expected_logging_operation
```
Comment (bkeryan) [inline tests/acceptance/test_internationalization.py:108]: The work item said to write component tests. I wasn't expecting more i18n acceptance tests, but I don't mind adding them.

Please do the following:
- Copy these tests to tests/component/task/test_in_stream.py and remove the i18n stuff from that copy (use an ASCII filename)
- Remove the non-i18n stuff from this copy (mode, group name, operation)

**Review by bkeryan — CHANGES_REQUESTED** [review]

File: tests/acceptance/test_internationalization.py (hunk 2 above)
Comment (texasaggie97) [reply tests/acceptance/test_internationalization.py]: Changed

**Review by bkeryan — APPROVED** [review]

File: tests/acceptance/test_internationalization.py (hunk 1 above)
Comment (texasaggie97) [reply tests/acceptance/test_internationalization.py:75]: Internal Work Item #2805031

**Pull request merged**